src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4680,4 +4680,13 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_PeWritingFailure" xml:space="preserve">
     <value>An error occurred while writing the output file: {0}</value>
   </data>
+  <data name="WRN_AttributeIgnoredWhenPublicSigning" xml:space="preserve">
+    <value>Attribute '{0}' is ignored when public signing is specified.</value>
+  </data>
+  <data name="WRN_AttributeIgnoredWhenPublicSigning_Title" xml:space="preserve">
```

Comment:
Each non-error severity compiler diagnostic needs a title. This is because when you open any ruleset file for a managed project, we show all the non-error compiler diagnostics in the ruleset editor, and the title for each diagnostic is displayed. See the below screenshot:

![compiler_diags_ruleset](https://cloud.githubusercontent.com/assets/10605811/15334588/b871cec6-1c23-11e6-8f3c-30b9f34a7475.png)